ipynb/examples/energy_meter/EnergyMeter_ACME.ipynb

```diff
@@ -92,10 +92,10 @@
     "        \"conf\" : {\n",
     "            #'iio-capture' : '/usr/bin/iio-capture',\n",
     "            #'ip_address' : 'baylibre-acme.local',\n",
-    "            'channels' : {\n",
-    "                'Device0' : 0,\n",
-    "                'Device1' : 1,\n",
-    "            }\n",
+    "        },\n",
```

Comment:
Oh not, right... we moved the "channel_map" to be an attribute of "emeter"... should be ok!